fe/fe-core/src/main/java/org/apache/doris/plsql/functions/InMemoryFunctionRegistry.java

```diff
@@ -251,6 +252,20 @@ public void addUserProcedure(Create_procedure_stmtContext ctx) {
         procMap.put(procedureName.toString(), ctx);
     }
 
+    @Override
+    public void removeUserProcedure(Drop_procedure_stmtContext ctx) {
+        FuncNameInfo procedureName = new FuncNameInfo(
```

Comment:
```
FuncNameInfo procedureName = new FuncNameInfo(
                exec.logicalPlanBuilder.visitMultipartIdentifier(ctx.multipartIdentifier()));
        if (builtinFunctions.exists(procedureName.toString())) {
            exec.info(ctx, procedureName.toString() + " is a built-in function which cannot be removed.");
            return;
        }
        if (trace) {
            trace(ctx, "DROP PROCEDURE " + procedureName.toString());
        }
```
Commonly used in `DorisFunctionRegistry.removeUserProcedure` and `InMemoryFunctionRegistry.removeUserProcedure`.

Consider moving to `visitDrop_procedure_stmt`